OpenRA.Mods.Common/Effects/Missile.cs

```diff
@@ -35,10 +35,10 @@ public class MissileInfo : IProjectileInfo
 		public readonly bool Shadow = false;
 
 		[Desc("Minimum vertical launch angle (pitch).")]
-		public readonly WAngle MinimumLaunchAngle = WAngle.Zero;
+		public readonly WAngle MinimumLaunchAngle = new WAngle(-64);
 
 		[Desc("Maximum vertical launch angle (pitch).")]
-		public readonly WAngle MaximumLaunchAngle = new WAngle(64);
+		public readonly WAngle MaximumLaunchAngle = new WAngle(128);
```

Comment:
I'd rather leave the default at 128 here and set it to something larger for the AGT's missile yaml rules specifically.